web/src/main/frontend/src/components/ui/Checkbox.js

```diff
@@ -15,6 +15,8 @@ const Checkbox = props => {
     }
   }
 
+  propsContainer.defaultChecked = props.defaultChecked;
```

Comment:
I dislike this entire component. Can you refactor it to be more in line with `mobx-react-form`:

```js
// example usage
<Checkbox {...form.$('whatever').bind()} someProp="1" />
```

Reply:
```suggestion
  <Checkbox
        answer = {answer}
        key = {option.id}
        option={option}
      />
```
Would something like this be better? Checkbox's code would look like:

```suggestion
<div className="checkbox_wrapper">
      <label>
        <input {...props.answer.bind(
          {
            value: props.option.id
            })} 
            defaultChecked={checked} 
            type="checkbox" />
        <div className="label_icon">
          <span className={answerTextClassName}>{props.option.description}</span>
        </div>
      </label>
    </div>
```

Reply:
Not sure I understand what I need to do.

Reply:
I'd like the checkbox component to behave as much as eg. a simple input would:

```js
// https://foxhound87.github.io/mobx-react-form/docs/quick-start.html
<input {...form.$('whatever').bind()} />
```

Reply:
Upon looking further at the code, I realized that CheckBox was used by two different forms and that their functionality was relatively different. I decided to remove the component entirely as it was not doing much already and making it work with different form types would require either adding an addiotional component type that would be used only once or making some ugly code to make it comply with everything.